department-of-veterans-affairs/abd-vro
# New positive end to end test

## What was the problem?


No end to end test that got data from both mas and lighthouse

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


There is a new end to end test. For collection 375 there will be mock mas data with blood pressures and medications and for lighhouse data from using a sandbox data.

## How to test this PR
- Run the end test.



[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
